src/components/Inputs/CorInputSwitch/CorInputSwitch.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { Meta, StoryFn } from '@storybook/vue3';
+import CorInputSwitch from './CorInputSwitch';
+
+export default {
+    title: 'Cornellius/Inputs/Input Switch',
+    component: CorInputSwitch,
+    argTypes: {
+        label: { type: 'string' },
+        disabled: { type: 'boolean' },
+        checked: { type: 'boolean' },
+        labelPosition: { control: 'select', options: ['left', 'right'] },
+    },
+} as Meta;
+
+const Template: StoryFn = (args) => <CorInputSwitch {...args} />;
```

Comment:
We have to be coherent here. Either we do `{...args}` or we keep props the way you did with InputTime.